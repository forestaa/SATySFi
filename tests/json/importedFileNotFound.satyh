@import: notfound

let x = 1
