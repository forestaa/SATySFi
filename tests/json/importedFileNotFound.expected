{
  "file": "",
  "range": {
    "start": { "line": 0, "character": 0 },
    "end": { "line": 0, "character": 0 }
  },
  "message":
    "! [Error] imported file not found:\n
notfound\n    candidate paths:\n      /mnt/c/Users/tdaic/build/SATySFi/tests/json/notfound.satyh\n      /mnt/c/Users/tdaic/build/SATySFi/tests/json/notfound.satyg"
}
